index.html

```diff
@@ -354,10 +354,10 @@ <h2>
             I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
         </p>
         <div>
-            <form>
-                <input class="hover" placeholder="Full name">
-                <input class="hover" placeholder="Email address">
-                <textarea class="hover" placeholder="Enter text here"></textarea>
+            <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure to create your unique `{form_id}` in order to be able to submit your form successfully. Please go to [formspree site](https://formspree.io/) how to use the endpoints for the action attribute on your form.